extensions/Worker.Extensions.Http.AspNetCore/src/Worker.Extensions.Http.AspNetCore.csproj

```diff
@@ -6,7 +6,8 @@
     <Description>ASP.NET Core extensions for .NET isolated functions</Description>
 
     <!--Version information-->
-    <VersionPrefix>1.3.3</VersionPrefix>
+    <VersionPrefix>2.0.0</VersionPrefix>
+    <VersionSuffix>-preview1</VersionSuffix>
     <TargetFramework>net6.0</TargetFramework>
```

Comment:
Should the TFM also be updated to reflect use of the newer types?

Reply:
I don't want to combine that change with this, but that will likely change for 2.0. Let's discuss whether we want that change in place before the first drop.